docs/examples/mnist.md

```diff
@@ -1,114 +1,239 @@
-This is a complete example of training an image classification model on the MNIST
+This is a complete example of training an image classification model on the MNIST handwritten digit
 dataset.
 
+These interactive notebooks follow the steps of this example:
+
+**TODO: point notebook URL to ludwig-ai/ludwig-docs repo before PR merge**
+
+- Ludwig CLI: [![MNIST Image Classification with Ludwig CLI](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/jimthompson5802/ludwig-docs/blob/expanded-mnist-example-with-colab/docs/examples/mnist_colab_notebooks/MNIST_Classification_with_Ludwig_CLI.ipynb)
+- Ludwig Python API: [![MNIST Image Classification with Ludwig API](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/jimthompson5802/ludwig-docs/blob/expanded-mnist-example-with-colab/docs/examples/mnist_colab_notebooks/MNIST_Classification_with_Ludwig_API.ipynb)
+
 ## Download the MNIST dataset
 
-```
-git clone https://github.com/myleott/mnist_png.git
-cd mnist_png/
-tar -xf mnist_png.tar.gz
-cd mnist_png/
+[MNIST](http://yann.lecun.com/exdb/mnist/) is a collection of gray-scale images of handwritten digits. This collection is made up of 60,000 images for training and 10,000 images for testing model performance.  Each image is 28 X 28 pixels in gray-scale.
+
+```shell
+ludwig datasets download mnist
 ```
 
-## Create train and test CSVs
+This command will create a dataset `mnist_dataset.csv` in the current directory.
```

Comment:
Maybe also mention that `ludwig datasets download mnist` creates `training/` and `testing/` directories in the current dir with the dataset images.

Reply:
done